### ISSUE TITLE
如何对模型里的参数进行手动更新

### ISSUE THREAD

**Issue by bixiaopeng0** [PaddlePaddle/Paddle#47218]
### 请提出你的问题 Please ask your question

                print("begin=>",self.model.state_dict()[k])
                if paddle.is_floating_point(v):
                    print("using ema deacy",decay)
                    self.model.state_dict()[k] *= decay
                    self.model.state_dict()[k] += (1.0 - decay) * msd[k].detach()
                    print("msd[k]",msd[k])
                    print("after=>",self.model.state_dict()[k])
采用上述方法，begin打印出的参数和after打印出的参数一样，self.model里的参数没有得到修改，请问应该怎么操作才能手动修改模型参数呢？

**Comment by zhangbo9674** [comment]
你好，建议通过paddle.save对修改后的state_dict重新保存，覆盖原文件：paddle.save(layer_state_dict, "layer.pdparams")

**Comment by bixiaopeng0** [comment]
`    @paddle.no_grad()
    def _momentum_update_key_encoder(self):
        """
        Momentum update of the key encoder
        """
        for param_q, param_k in zip(self.encoder_q.parameters(),
                                    self.encoder_k.parameters()):
            paddle.assign((param_k * self.m + param_q * (1. - self.m)), param_k)
            param_k.stop_gradient = True`
